iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/Tools.java

```diff
@@ -92,7 +92,7 @@ public static String retrieveEnvironmentVariableValue(String environmentVariable
         }
         catch (ClassNotFoundException e)
         {
-            log.debug("Likely running the JVM tests, ignoring ClassNotFoundException");
+            log.debug("Likely running the JVM tests, ignoring ClassNotFoundException\n");
```

Comment:
For some reason, this log never had a newline character printed even though you don't normally have to manually add one. I went ahead and added so it definitely would

Reply:
interesting that we don't add this elsewhere but it still adds a newline at the end of each log